pkg/agent/core/state_test.go

```diff
@@ -1418,7 +1415,23 @@ func TestMetricsConcurrentUpdatedDuringDownscale(t *testing.T) {
 	}
 
 	// Send initial scheduler request - without the monitor active, so we're stuck at 4 CU for now
-	doInitialPluginRequest(a, state, clock, duration("0.1s"), DefaultComputeUnit, nil, resForCU(4))
+	a.
```

Comment:
I guess this test is not directly related to this PR but it's fine.

Reply:
Ah, right - I did this because the expected warnings changed here. But looking at this again, just realized that I can actually just pass `a.WithWarnings(...)` into `doInitialPluginRequest`, which should make this simpler.